crates/nostr/src/types/contact.rs

```diff
@@ -3,6 +3,7 @@
 
 //! Contact
 
+use crate::event::tag::UncheckedUrl;
```

Comment:
In my crates I import modules in this order:
- std modules
- external crates
- local modules

To avoid that `cargo fmt` change the order, leave a space between these lines:

```rust
use secp256k1::XOnlyPublicKey;

use crate::event::tag::UncheckedUrl;
```

It's just to keep the code cleaned and well readable :)